src/components/panes/ConfirmImportPane.jsx

```diff
@@ -25,17 +26,17 @@ export default class ConfirmImportPane extends PaneBase {
 
     getFieldID(item) {
         if (item.data.type == "id") {
-            return item.data.config.origin;
+            return { type: item.data.config.origin, name: item.data.config.origin };
         } else if (item.data.type == "person_data") {
-            return item.data.config.field;
+            return { type: item.data.config.field, name: item.data.config.field };
         } else if (item.data.type == "person_tag") {
-            return item.data.type;
+            return { type: item.data.type };
         } else if (item.data.type == "person_field") {
             const field = this.props.fieldTypes.items.find((f) => 
                 f.data.id == item.data.config.field_id);
-            return field.data.type + '.' + item.data.config.field_id;
+            return { type: field.data.type, name: 'field.' + item.data.config.field_id };
         } else {
-            return "unknown_type";
+            return { type: "unknown_type" };
```

Comment:
Nice solution! Would you say that this fixes #1128?

Reply:
Yes it does